flow/activities/flowable.go

```diff
@@ -638,6 +636,7 @@ func (a *FlowableActivity) replicateQRepPartition(ctx context.Context,
 	var stream *model.QRecordStream
 	bufferSize := shared.FetchAndChannelSize
 	errGroup, errCtx := errgroup.WithContext(ctx)
```

Comment:
This can now move into `config.SourcePeer.Type == protos.DBType_POSTGRES` branch